.idea/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Default ignored files
```

Comment:
This shouldn’t be included.

Reply:
Editor-specific files were excluded before every commit. I think I've missed that. So I've removed it now. Also, I've added a `.gitignore` file to the top level. Is there any reason that you didn't add this before?